include/picongpu/fields/laserProfiles/ExpRampWithPrepulse.def

```diff
@@ -0,0 +1,177 @@
+/* Copyright 2018 Ilja Goethel, Axel Huebl
+ *
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include <pmacc/types.hpp>
+
+
+namespace picongpu
+{
+namespace fields
+{
+namespace laserProfiles
+{
+namespace expRampWithPrepulse
+{
+namespace defaults
+{
+    struct ExpRampWithPrepulseParam
+    {
+        // Intensities of prepulse and exponential preramp
+        static constexpr float_X INT_RATIO_PREPULSE = 0.;
+        static constexpr float_X INT_RATIO_POINT_1 = 1.e-8;
+        static constexpr float_X INT_RATIO_POINT_2 = 1.e-4;
+        static constexpr float_X INT_RATIO_POINT_3 = 1.e-4;
+
+        // time-positions of prepulse and preramps points
+        static constexpr float_64 TIME_PREPULSE_SI = -950.0e-15;
+        static constexpr float_64 TIME_PEAKPULSE_SI = 0.0e-15;
+        static constexpr float_64 TIME_POINT_1_SI = -1000.0e-15;
+        static constexpr float_64 TIME_POINT_2_SI = -300.0e-15;
+        static constexpr float_64 TIME_POINT_3_SI = -100.0e-15;
+
+        /** unit: meter */
+        static constexpr float_64 WAVE_LENGTH_SI = 0.8e-6;
+
+        /** UNITCONV */
+        static constexpr float_64 UNITCONV_A0_to_Amplitude_SI = -2.0 * PI / WAVE_LENGTH_SI * ::picongpu::SI::ELECTRON_MASS_SI * ::picongpu::SI::SPEED_OF_LIGHT_SI * ::picongpu::SI::SPEED_OF_LIGHT_SI / ::picongpu::SI::ELECTRON_CHARGE_SI;
+
+        /** unit: W / m^2 */
+        // calculate: _A0 = 8.549297e-6 * sqrt( Intensity[W/m^2] ) * wavelength[m] (linearly polarized)
+
+        /** unit: none */
+        static constexpr float_64 _A0  = 20.;
+
+        /** unit: Volt /meter */
+        static constexpr float_64 AMPLITUDE_SI = _A0 * UNITCONV_A0_to_Amplitude_SI;
+
+        /** unit: Volt /meter */
+        //constexpr float_64 AMPLITUDE_SI = 1.738e13;
```

Comment:
@ax3l This is equal to an a_0 of 4.33 but above a a_0 of 20 is used. Should we make this consistent with this pull request or do we leave it as is/as it was?

Reply:
If you like we can make them consistent in a follow-up.